doc/user/content/quickstarts/abandoned-cart.md

```diff
@@ -0,0 +1,333 @@
+            (regexp_match(data->>'url', '/(?:products|profiles)/(\d+)')[1])::int AS target_id,
+            data->>'channel' AS channel,
+            (data->>'received_at')::bigint AS received_at
+        FROM (SELECT CONVERT_FROM(data, 'utf8')::jsonb AS data FROM json_pageviews);
+    ```
+
+1. Define a view containing the incomplete orders:
+
+    ```sql
+    CREATE VIEW incomplate_purchases AS
+        SELECT
+            users.id AS user_id,
+            users.email AS email,
+            users.is_vip AS is_vip,
+            purchases.item_id,
+            purchases.status,
+            purchases.quantity,
+            purchases.purchase_price,
+            purchases.created_at,
+            purchases.updated_at
+        FROM users
+        JOIN purchases ON purchases.user_id = users.id
+        WHERE purchases.status = 0;
+    ```
+
+1. Create a [`MATERIALIZED VIEW`](/sql/create-materialized-view/) to get all the users that are no longer browsing the site.
+
+    A `MATERIALIZED VIEW` is persisted in a durable storage and is incrementally updated as new data arrives.
+
+    ```sql
+    CREATE MATERIALIZED VIEW inactive_users_last_3_mins AS
+        SELECT
+            user_id,
+            date_trunc('minute', to_timestamp(received_at)) as visited_at_minute
+        FROM v_pageviews
+        WHERE mz_now() >= (received_at*1000 + 180000)::numeric
+        GROUP BY 1,2;
+    ```
+
+    The above view uses a [temporal filter](/sql/patterns/temporal-filters/) to only include users who have not visited the site in the last 3 minutes.
+
+1. Create a materialized view to join the incomplete purchases with the inactive users to get the abandoned carts:
+
+    ```sql
+    CREATE MATERIALIZED VIEW abandoned_cart AS
+        SELECT
+            incomplate_purchases.user_id,
+            incomplate_purchases.email,
+            incomplate_purchases.item_id,
+            incomplate_purchases.purchase_price,
+            incomplate_purchases.status
+        FROM incomplate_purchases
+        JOIN inactive_users_last_3_mins
+            ON inactive_users_last_3_mins.user_id = incomplate_purchases.user_id
+        GROUP BY 1, 2, 3, 4, 5;
+    ```
+
+1. Select from the `abandoned_cart` view to see the results:
+
+    ```sql
+    SELECT * FROM abandoned_cart LIMIT 10;
+    ```
+
+1. Use [`SUBSCRIBE`](/sql/subscribe) to the `abandoned_cart` view to see the results in real-time:
+
+    ```sql
+    COPY ( SUBSCRIBE ( SELECT * FROM abandoned_cart ) ) TO STDOUT;
+    ```
+
+    You can use that real-time data to send an email to the user or send a push notification.
+
+## Send notifications to Telegram
+
+Materialize is wire-compatible with PostgreSQL, so you can use any PostgreSQL client to connect to Materialize. In this example, we'll use the [Node.js](/integrations/node-js) and create a simple script that sends a message to a Telegram user when a user abandons a cart in real time.
+
+### Create a Telegram bot
```

Comment:
What do you think about having the Telegram-specific instructions in the `/demos` repo, but not here? Since it requires users to install something else, we could phrase this section more as a reference example and just include the code snippet where the bot interacts with Materialize (and add a GIF of a message being received or something visual like that).

Reply:
I've moved that section to the demos repo: https://github.com/MaterializeInc/demos/pull/54

I think that if we only show the code without giving the extra details, or just limited details, it might be confusing for the readers, especially if they've not used Telegram specifically. Maybe we just give some references to the `/connection-examples` repo or maybe instead describe how they could use a Kafka sink (even though they will not be able to do so with the provided public kafka cluster). let me know what you think!